app/features-json/project_json_controller.rb

```diff
@@ -9,8 +10,11 @@ class ProjectJSONController < AuthenticatedControllerBase
       current_provider_credential = check_and_get_provider_credential
       current_user_config_service = self.current_user_config_service
       all_projects = current_user_config_service.projects(provider_credential: current_provider_credential)
+      all_projects_views_models = all_projects.map do |project|
+        ProjectSummaryViewModel.viewmodel_from!(object: project)
```

Comment:
I think we can do this in a single line:

```ruby
.map(&ProjectSummaryViewModel.method(: viewmodel_from!))
```

Reply:
nice!